app/1.0/toolbox/news-case-study.md

```diff
@@ -0,0 +1,264 @@
+    else:
+      self.response.out.write(open('index.html').read())
+```
+
+## Resource URLs
+
+Resource URLs, like `/bower_components/`, `/data/` and `/src/` are served as static files. These static URLs are defined in `app.yaml`. These URLs are used internally by the app to load components and data.
+
+## Displaying Ads
+
+The News app attempts to display ads, without blocking a fast first paint.
+
+The <news-iframe> element uses [`IntersectionObserver`](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API), a web platform API. `IntersectionObserver` is used to determine whether `<news-iframe>` element is visible on screen before creating an iframe that will display an ad.
+
+`news-iframe.html`
+```javascript
+attached: function() {
+  if ('IntersectionObserver' in window) {
+    this._io = new IntersectionObserver(this._createIframe.bind(this));
+    this._io.observe(this);
+  } else {
+    // Intersection Observer is not available, so just create the iframe
+    // after a bit of a delay.
+    window.setTimeout(this._createIframe.bind(this), 100);
+  }
+},
+```
+
+If `IntersectionObserver` isn't available, the application briefly delays displaying the ad to let the rest of the page load first.
+
+## AMP version of the News app
+
+We built a [version of the News app](https://github.com/PolymerLabs/news/tree/amp) for integration with [AMP (Accelerated Mobile Pages)](https://www.ampproject.org/) documents. In the AMP version of News, the HTML content files (for example, `/data/articles/it-takes-teacher.html`) are replaced by AMP documents which can be loaded as standalone pages.
+
+The AMP project supports adding an AMP document to a shadow root on your page. Here’s an example: <code>[https://github.com/ampproject/amphtml/blob/master/examples/pwa/pwa.js](https://github.com/ampproject/amphtml/blob/master/examples/pwa/pwa.js)</code>
+
+To support this in the AMP version of the News app, we created [the `<amp-viewer>` element](https://github.com/PolymerLabs/amp-viewer/). The `<amp-viewer>` element works similarly to an iframe. To load an AMP document inside the viewer, set its `src` attribute to the URL of the AMP document. Styles will be applied to the content, scoped by its shadow root.
+
+The element is written as a native custom element, and can be used independently of the News app.
+
+ Here's how the `<amp-viewer>` element is used in News:
+
+`news-article.html`
+```html
+<div class="container" fade-in$="[[!loading]]" hidden$="[[failure]]">
+  <article class="flex">
+    <amp-viewer id="ampViewer" src="[[_getArticleSrc(article.id)]]"></amp-viewer>
+  </article>
+```
+
+Note: the AMP version of the app does not support swiping between articles - doing so would require loading the title, image and author of the previous or next article. In the AMP version of News, that data is only loaded when the article content is loaded. It is not loaded with the list of articles.
+
+## More Resources
```

Comment:
I think to highlight the other docs, this should be, "Docs and more resources" maybe (?) and add a link to the microsite, like, "For documentation on using and customizing the News app ..."